app/proguard-rules.pro

```diff
@@ -41,3 +41,5 @@
 
 # Dagger
 -dontwarn com.google.errorprone.annotations.*
+
+-dontwarn com.google.firebase.perf.network.FirebasePerfUrlConnection
```

Comment:
I had to add this, if not when I ran `./gradlew clean assemble bundle` I got this error:

```
ERROR: Missing classes detected while running R8. Please add the missing classes or apply additional keep rules that are generated in /Users/javierarroyo/Projects/Personal/Forlago/app/build/outputs/mapping/release/missing_rules.txt.
ERROR: R8: Missing class com.google.firebase.perf.network.FirebasePerfUrlConnection (referenced from: java.util.List kotlinx.coroutines.internal.FastServiceLoader.parse(java.net.URL))
```

Reply:
Something feels wrong here: we don't have such rule in the Link proguard files.